demo/packages/tabs/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,229 @@
+🤔 Imagine these components offer great extendability and flexibility
+
+Now stop imagining, and look at lion 🦁
+
+## What is lion?
+
+> We want to bring the web forward - one component at a time.
+
+Lion is a white label, open source, framework agnostic component library, and can  be the foundation for your codified in-house Design System. Aligning on design and usability is already a tough challenge, with Lion, we hope to take some of the work out of your hands.
+
+### White Label
+Lion is a core package of white label Web Components. What this means is that the core components only consist of very minimal styling, yet has all the fundamental functionalities. White label products are often created so others can easily rebrand them with their own visual identity. This is great, because it means that everyone can use the core functionalities of our components, while bringing their own branding or design system, because surprisingly, not everyone loves orange.
+
+Check out the [Lion demo's](http://lion-web-components.netlify.com/). Looks plain, doesn't it?
+
+And this is exactly what we at ING do as well. Our ing-web components extend the lion components, and apply our own ING visual identity which mostly is a thin layer on top of lion.
+
+Let's take a look at how Lion and ing-web compare to eachother:
+
+<<---- ADD IMAGE ----->>
+
+### 🎯 Focus
+Lion was designed with a focus on global UX and reusability in mind. And as such, the following features were incorporated right from the start:
+
+♻️ Reusability - Our components are meant to be distributed and used on a global scale
+🚹 Accessibility - Our components are meant to be accessible for everyone
+🚀 Performance - Our components are meant to be small, performant and fast
+
+These features allows us to roll out our components globally and have a single point of truth, and ensure that everyone in ING has a solid set of building blocks to build their application with, and get up and running in no time.
+
+### Lessons learned
+
+ING started using web components very early on, and Lion is not the first component library we built. So you may wonder, how are these components different from the previous generation?
+
+Lion is built from the ground up allow for accessibility and extendability as we learned that these things are almost impossible to change/achieve at a later point. We'd like to highlight a few of the lessons learned while making the new set of components.
+
+- Not everything has to be a Web Component, for certain functionality regular JavaScript is better.
+- Stay as close to native semantic HTML elements as possible.
+- Incorporate accessibility from the start.
+- Not everything needs to be in the shadow DOM, this is especially important for aria relations and accessibility.
+- UI components are hard
+
+> Everyone should use web components but not everyone should write them
+
+*Erik*
+
+## 🙋‍ Looking For Contributors/Adopters!
```

Comment:
This may or may not be desirable, but perhaps it's worth setting up a public Slack/whatever space for Lion adopters to chat about it, like we do have on Mattermost.

Reply:
we already have 👍  but yes we should link to it here 🤗